File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/Commands/DefinePosicao.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class DefinePosicao(private val posicao: Int) : TermicaCommand("DefinePosicao") {
    override fun functionParameters(): String {
        return "\"posicao\"" + ":" + posicao
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/define_posicao.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class DefinePosicao extends TermicaCommand {
  final int _posicao;

  DefinePosicao(this._posicao) : super('DefinePosicao');

  @override
  functionParameters() => '"posicao"' ':' '$_posicao';
}
